pkgs/top-level/all-packages.nix

```diff
@@ -5670,6 +5670,8 @@ with pkgs;
 
   hopper = qt5.callPackage ../development/tools/analysis/hopper { };
 
+  howdy = callPackage ../tools/system/howdy { };
```

Comment:
New packages are recommended to follow [RFC 140](https://github.com/NixOS/rfcs/blob/master/rfcs/0140-simple-package-paths.md). To do this, move `pkgs/tools/system/howdy/default.nix` to `pkgs/by-name/ho/howdy/package.nix` and delete this line.

Reply:
fixed in https://github.com/NixOS/nixpkgs/pull/254359/commits/a380ae22eb402f65415e278e41f0e3dc8bd20b5f